Cargo.toml

```diff
@@ -41,6 +41,10 @@ name = "cmd-program"
 name = "colored"
 required-features = ["colored"]
 
+[[example]]
+name = "pretty-colored"
+required-features = ["colored"]
+
```

Comment:
A small link just mentioning "For full-line colors, see [this example](github.com/daboross/fern/blob/master/...)" to the docs in `src/colors.rs` would help give this example some more visibility if you want.

Reply:
okay, do you want to do that? I'm not really sure where to place it.

Reply:
Sure, I can do that.